test/samples/async-await.js

```diff
@@ -0,0 +1,25 @@
+module.exports = [
+	{
+		description: 'transpiles await arrow function call',
+		input: `async () => await a()`,
+		output: `!function() { return Promise.resolve().then(function() { return a() }); }`
```

Comment:
What's the point of `!function`? Just to draw attention that it's not being used?

Reply:
It turns it from a function declaration into a function expression, see: https://stackoverflow.com/questions/3755606/what-does-the-exclamation-mark-do-before-the-function

As for why I'm not entirely sure.

Reply:
The input is a function expression, too.